### DISCUSSION TITLE
v1.1.5

### DISCUSSION THREAD

**Discussion by bananarama92** [bananarama92/MBS#66] [Announcements]
* Fix a faulty `TypeRecord` check

## What's Changed
* REL: Prepare for 1.1.5 release by @bananarama92 in https://github.com/bananarama92/MBS/pull/65

**Full Changelog**: https://github.com/bananarama92/MBS/compare/v1.1.4...v1.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/bananarama92/MBS/releases/tag/v1.1.5'>v1.1.5</a>.</em>

**Comment by anchillabc** [comment]
Trying to get the "quickest time from report to fix" achievement?
